File: ApiSuvesaPos/Negocio/Modelo/cxc/firmadocontado/valida_firmadocontado.vb
Namespace Modelo
    Public Class valida_firmadocontado
        Public Property ID_VALIDA_FIRMADOCONTADO As Integer
        Public Property CONTADO As Boolean
        Public Property PVE As Boolean
        Public Property MONTO_MAXIMO As Integer
        Public Property EXIGE_NOMBRE As Boolean
        Public Property MAXIMO_CLIENTE As Integer
        Public Property MAXIMO_AUTORIZA As Integer
        Public Property MAXIMO_RETIRA As Integer
        Public Property Exepcion_FirmadoContado As New List(Of Modelo.exepcion_firmadocontado)
    End Class
End Namespace

File: ApiSuvesaPos/ApiSuvesaPos/Modelo/cxc/firmadocontado/valida_firmadocontado.cs
using System.Collections.Generic;

namespace Modelo
{
    public class valida_firmadocontado
    {
        public int ID_VALIDA_FIRMADOCONTADO { get; set; }
        public bool CONTADO { get; set; }
        public bool PVE { get; set; }
        public int MONTO_MAXIMO { get; set; }
        public bool EXIGE_NOMBRE { get; set; }
        public int MAXIMO_CLIENTE { get; set; }
        public int MAXIMO_AUTORIZA { get; set; }
        public int MAXIMO_RETIRA { get; set; }
        public List<Modelo.exepcion_firmadocontado> Exepcion_FirmadoContado { get; set; } = new List<Modelo.exepcion_firmadocontado>();
    }
}
